contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,189 @@
+---
+date: 2020-08-18
+title: Why You May Not Need a Sales Team
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+
+<!-- ![Segment Banner](../images/plg-header.png) -->
+
+Let's say you are the founder of a new tech startup. 
+
+A few months back, you and a couple of friends (all developers) built an amazing tool and have now received some Seed funding. 
+
+As a result, you're under some pressure to grow. You took a few million from some VCs here and there, and they want to see some results.
+
+Since the realization of needing to grow hit you, you've started to lose some sleep. 
+
+Before, you were just a group of friends working your ass off to build an amazing product. Now, you actually have to focus on selling it too.
```

Comment:
this is kind of unrealistic for 99% of companies - I'd expect them to have some revenue.

Maybe "working your ass off to build an amazing product, hopefully with its first revenue starting to come in - which you got through friends of friends or just blind luck. Now you actually have to focus on ramping up your sales."